lib/core/db_models/old/schema/table_collection_schema.dart

```diff
@@ -26,4 +26,5 @@ class FbCollections {
   static const String checks = "checks";
   static const String occupations = "occupations";
   static const String workHistories = "work_histories";
+  static const String workerNotificationMessage = "worker_notification_messages";
```

Comment:
guessing this is a mistake and we shouldn't have an underscore between these words

Reply:
sam this is the collection name and it should be same as mentioned in the firestore